letsencrypt/display/util.py

```diff
@@ -61,10 +75,11 @@ def menu(self, message, choices,
 
         :param str ok_label: label of the OK button
         :param str help_label: label of the help button
+        :param dict _kwargs: absorbs default / cli_args
 
-        :returns: tuple of the form (`code`, `tag`) where
```

Comment:
nit: Why were the backticks removed? They provide formatting to the Sphinx documentation. This removal was done in a number of places.